internal/provider/resource/integration_aws_secrets_manager.go

```diff
@@ -435,6 +448,13 @@ func (r *IntegrationAWSSecretsManagerResource) Update(ctx context.Context, req r
 		return
 	}
 
+	plan.Options = plan.Options
+	plan.SecretPath = types.StringValue(updatedIntegration.Integration.SecretPath)
+	plan.IntegrationAuthID = types.StringValue(updatedIntegration.Integration.IntegrationAuthID)
+	plan.Environment = types.StringValue(updatedIntegration.Integration.Environment.Slug)
+
+	diags = resp.State.Set(ctx, plan)
```

Comment:
we're doing this twice
<img width="808" alt="image" src="https://github.com/user-attachments/assets/2bd0a867-35b4-406f-aa49-d792eeb35123">